### PR TITLE
Add no supplier events

### PR DESCRIPTION
**Pull request by antonin-francois**

### CODE REVIEW

**Comment by antonin-francois** [comment]
* **#51** <a href="https://app.graphite.dev/github/pr/miamtech/MealzSharedAnalytics/51?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @antonin-francois and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by diarmuidmcg — APPROVED** [review]

File: mealz-shared-analytics/src/commonMain/kotlin/mealz/ai/AbstractSharedAnalytics.kt [hunk 1]
```diff
@@ -30,7 +30,7 @@ abstract class AbstractSharedAnalytics {
         // an empty path (mobile) so we assume it is valid
         if (!path.contains("/miam/")) return
 
-        val validParts = "||miam|recipes|liked|categories|my-meals|detail|replace-item|sponsor|meals-planner|catalog|results|basket-preview|finalize|"
+        val validParts = "||miam|recipes|liked|categories|my-meals|detail|replace-item|sponsor|meals-planner|catalog|results|basket-preview|finalize|onboarding|locator|"
```
Comment (ThibaultMiam) [inline mealz-shared-analytics/src/commonMain/kotlin/mealz/ai/AbstractSharedAnalytics.kt:33]: pk tu as deux pipe en debut de chaine ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
-# 1.0.0
+## 2.0.0
+[BRK][FIX] Rename search.store in pos.search
```
[the remark this answers is not in the document]
Comment (antonin-francois) [reply CHANGELOG.md:2]: pas grave, si ?

**Review by ThibaultMiam — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ThibaultMiam) [reply CHANGELOG.md:2]: non mais elle a pas fait long feu la V1 ^^

**Pull request merged**